restructure metrics and server-side sampling content (#5479)
getsentry/sentry-docs: src/docs/product/sentry-basics/search/searchable-properties.mdx

When searching on event properties within the **Issues** page, the search will return any issue that has _one or more events_ matching the supplied event properties filter.

> In the case of transaction events, we also refer to these types of properties as _indexed event properties_ because they are transaction events for which we store all of the data. This isn't the case for [processed event properties](#processed-event-properties). **For error events, we currently store all data for all events sent to Sentry, and the concept of processed events doesn't apply.** Learn more about indexed and processed transaction events in our full [Sampling documentation](/product/sentry-basics/sampling/).

Author: ```suggestion
In the case of transaction events, we also refer to these properties as _indexed event properties_ because they are based on transaction events for which we store all of the data. This isn't the case for [processed event properties](#processed-event-properties). **For error events, we currently store all data for all events sent to Sentry, and the concept of processed events doesn't apply.** Learn more about indexed and processed transaction events in our full [Sampling documentation](/product/sentry-basics/sampling/).
```

> In the case of transaction events, we also refer to these types of properties as _indexed event properties_ because they are transaction events for which we store all of the data. This isn't the case for [processed event properties](#processed-event-properties). **For error events, we currently store all data for all events sent to Sentry, and the concept of processed events doesn't apply.** Learn more about indexed and processed transaction events in our full [Sampling documentation](/product/sentry-basics/sampling/index/).

Author: ```suggestion
In the case of transaction events, we also refer to these types of properties as _indexed event properties_ because they are transaction events for which we store all of the data. This isn't the case for [processed event properties](#processed-event-properties). **For error events, we currently store all data for all events sent to Sentry, and the concept of processed events doesn't apply.** Learn more about indexed and processed transaction events in our full [Sampling documentation](/product/sentry-basics/sampling/).
```

### Processed Event Properties

<Note>

This feature is in beta and is only available if your organization is participating in its limited release. Features in beta are still in-progress and may have bugs. We recognize the irony. If you’re interested in participating, [join the waitlist](https://sentry.io/for/performance/#updates-signup).

</Note>

> Processed events only apply to transactions and they comprise all the transaction events that you send to Sentry. These events represent aggreate data, and underly metrics in the **Performance**, **Discover**, and **Dashboards** features. In **Dashboards**, for the ["Errors & Transactions"](/product/dashboards/widget-builder/#errors--transactions) dataset, we use this aggreate data to generate results if all search conditions and/or columns are processed event properties.

Author: ```suggestion
Processed events only apply to transactions and they comprise all the transaction events that you send to Sentry. These events represent aggregate data and underly metrics in the **Performance**, **Discover**, and **Dashboards** features. In **Dashboards**, for the ["Errors & Transactions"](/product/dashboards/widget-builder/#errors--transactions) dataset, we use this aggreate data to generate results if all search conditions and/or columns are processed event properties.
```
